html_app/master_model.js

```diff
@@ -13900,26 +13908,34 @@ var __scb_sample_3 = {
 					headings: ['','Strain', 'Treatment', 'Treatment Duration'],
                     'WT-EGFR': {
                         rows: [
-                        {   'cell_treatments': {   'gm30': [   {
-                            'cell_line': 'WT-EGFR',
-                            'treatment_list': {   'list': [   {
-                                'collection_id': '30 s',
-                                'drug_list': {   'list': [   {
-                                    'concentration_id': '100',
-                                    'drug_id': 'growth_media'}]},
-                                'duration': '30 sec',
-                                'duration_value': 30,
-                                'microscope': [   'g'],
-                                'temperature': '25'}]}}]},
-                            'cells': [   { 'kind': 'checkbox','name': 'gm30','treatment_id': 'growth_media'},
-                                {'kind': 'text', 'text': 'WT-EGFR'},
-                                {'kind': 'text', 'text': 'Growth media only'},
-                                {'kind': 'text', 'text': '30 sec'}],
-                            'treatment_id': 'growth_media'},
+                            {   'cell_treatments': {   'gm30': [
+                                {
+                                    'cell_line': 'WT-EGFR',
+                                    'treatment_list': {   'list': [
+                                        {
+                                            'collection_id': 'default',
```

Comment:
The microscopy parser is using collection_id to identify the image pool. In this case I use default because collections do not change the experiment outcome.